### PR TITLE
Copy static files into build public folder

### LINKED ISSUE #1132: Manifest.json is no longer built in the post razzle world
**Describe the bug**
After moving away from razzle in https://github.com/BBC-News/simorgh/pull/1117 it appears the manifest.json file is no longer generated

**To Reproduce**

1. Run npm run build && npm run start
2. Go to http://localhost:7080/manifest.json
3. See the file doesnt exist

**Expected behavior**
The file is generated and served

- [x] Initially labelled with ["bug"](https://github.com/BBC-News/simorgh/labels/bug)

### PR DESCRIPTION
**Pull request by dr3**
Resolves #1132

Copies static files into public folder as before

Automated tests will be written for this in https://github.com/bbc/simorgh/issues/1131

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval
- [ ] I have followed [the merging checklist](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/MERGE_PROCESS.md) and this is ready to merge.

### CODE REVIEW

File: webpack.config.client.js [hunk 1]
```diff
@@ -49,6 +50,13 @@ module.exports = ({ resolvePath, IS_CI, IS_PROD, START_DEV_SERVER }) => {
         path: resolvePath('build'),
         filename: 'assets.json',
       }),
+      // copy static files
```
Comment (ChrisBAshton) [inline webpack.config.client.js]:
```suggestion
      // copy static files otherwise untouched by Webpack, e.g. favicon 
```

**Review by ChrisBAshton — COMMENTED** [review]
Looks good, thanks Drew. Just made the comment slightly more informative

**Review by ChrisBAshton — DISMISSED** [review]

**Review by bcmn — DISMISSED** [review]
Looks great locally 👍

**Comment by dr3** [comment]
Had to update (correct) the asset copy as the service worker was confused about where they were and throwing errors.

**Comment by dr3** [comment]
`npm start` should work, however `npm run dev` wont find the SW.js file OR the static assets. This is because they are currently served on the wrong port. This will be fixed in https://github.com/bbc/simorgh/issues/1152 as it is out of scope for this issue.

**Review by bcmn — DISMISSED** [review]

**Review by ChrisBAshton — DISMISSED** [review]
The important thing is this'll fix it on Test/Live 👍

**Comment by jamesbrumpton** [comment]
LGTM 👍

**Review by ChrisBAshton — APPROVED** [review]

**Review by bcmn — APPROVED** [review]

**Pull request merged**